BioForm/profiles/views.py

```diff
@@ -8,7 +8,7 @@
 from . models import *
 from profiles.templates.forms import CreateUserForm, BioForm
 
-@login_required(login_url = 'login')
```

Comment:
If your middleware implementation is correct then these decorators should work properly and you will be able to access current active user by doing `request.user`.

Reply:
I have commented this out because I am now doing the authentication redirects with my custom middleware. I did not remove them for my own sake in case I needed to revise concepts in the future.